titan-server/src/accounts/acl.rs

```diff
@@ -3,23 +3,23 @@ use diesel::{prelude::*, MysqlConnection};
 use crate::models;
 use crate::schema;
 
-pub fn get_user_acl(user_id: i32, wcf_db: &MysqlConnection) -> QueryResult<Vec<models::WcfAclOption>> {
+pub fn get_user_acl(wcf_user_id: i32, wcf_db: &MysqlConnection) -> QueryResult<Vec<models::WcfAclOption>> {
```

Comment:
This is a good rename. Good catch on needing to clarify the ID being used to query for ACL.